### DISCUSSION TITLE
When 1.9 Could Release

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#266] [General]
Well, it's been over a month since 1.8.1 debuted, giving us a slew of new features. So now the question is, when will 1.9 debut, and what new features could we expect from it? So far, the largest gap between major releases is 3 months, so there's a possibility that we could get the next major update within the next 3 months.

**Comment by DinoAwakening** [comment]
Be patient. I think Keiwan is working real hard on it, choosing the ideas the community actually wants :DD

**Reply by DWrightAnimation** [reply]
I'm trying to be patient, but this app is so awesome, and Keiwan is awesome for making it. I hope it becomes an industry standard for those who work in the industry but don't have a computer or the money to buy Adobe Animate or Toon Boom Harmony, or even for those who find it tedious to sit at the computer all day when they could animate on the go.

**Reply by DinoAwakening** [reply]
I totally agree. Even if he wants it to worth 20$ by now. ToonSquid 2.0 sounds so promising, and such a great update full of tons features and plenty possibilities and options :DD

**Reply by DWrightAnimation** [reply]
Well, looks like my prediction was right. And not only that, but 1.9 added Shape Assist and the ability to export/import assets from/to different projects. Now, we just wait for 2.0, which will probably have even more awesome features.

**Comment by habahu** [comment]
It would be better to have a roadmap.

**Reply by DWrightAnimation** [reply]
Like on Trello, that would be a great way to get people hyped for the next version releases. I mean, Callipeg has one, what's stopping ToonSquid from having one?

**Reply by DinoAwakening** [reply]
I think it’s unnecessary (Or even a bad idea since it’s Only Keiwan). Since excitement could easily become desperation. Then people would demand to keiwando to either tell the launch date, or code quicker.
